dpc/src/traits/parameters.rs

```diff
@@ -108,10 +106,7 @@ pub trait Parameters: 'static + Sized + Send + Sync {
         + PartialEq
         + Eq
         + Send
-        + Sync
-        + CanonicalSerialize
-        + CanonicalDeserialize
-        + PartialEq;
+        + Sync;
```

Comment:
Note: DPC no longer uses CanonicalSerialize and CanonicalDeserialize for the account signing key.